src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutMissingTagCheckTest.java

```diff
@@ -19,6 +20,9 @@
     @Rule
     public ConsumerBasedExpectedCheckVerifier verifier = new ConsumerBasedExpectedCheckVerifier();
 
+    private final Configuration inlineConfiguration = ConfigurationResolver.inlineConfiguration(
+            "{\"RoundaboutMissingTagCheck\":{\"tags.filter\":\"motor_vehicle->!no&foot->!yes&footway->!&access->!private&construction->!\"}}");
```

Comment:
Can you change the configurable here too?

Reply:
fixed.